docs(fix): removed broken images MTA-1604 (#245)
scaleway/docs-content: dedibox-network/domains/quickstart.mdx

## How to purchase a domain

> 1. In your browser, navigate to [the domain name order page](https://www.scaleway.com/en/domain/) by clicking **DNS** in the **Network** section of the side menu in the Scaleway Console.

Reviewer: ```suggestion
1. Click **DNS** in the **Network** section of the console side menu to access the [the domain name order page](https://www.scaleway.com/en/domain/).
```
